File: updateScore.py
score = 0
color = "blue"

def updateScore(points):
    global score 
    global color
    score += points
    if score < 0:
        score = 0
    color = "green" if points > 0 else "red"

#testing    
#updateScore(3);
#print "Testing (3): ", score
#print "Testing (green): ", color
#updateScore(-6);
#print "Testing (0): ", score
#print "Testing (red): ", color

File: updateScore.js
var score = 0;
var color = "blue";

function updateScore(points){
	score += points;
	if(score < 0)
		score = 0;
	color = (score > 0) ? "green" : "red";
}

/*
//testing
updateScore(3);
alert("Test (3): " + score);
alert("Test (green): " + color);
updateScore(-6);
alert("Test (0): " + score);
alert("Test (red): " + color);
*/